### PR TITLE
feat: Add support for generating changelogs

### PR DESCRIPTION
**Pull request by halkeye**
* See https://artifacthub.io/docs/topics/annotations/helm/
* trim out any blank lines

You can see it working https://github.com/halkeye/test-semantic-release-helm3/tree/5a231415e801a4c38aab759b6031f77f69a8980c

### CODE REVIEW

**Review by nflaig — CHANGES_REQUESTED** [review]
Thanks @halkeye for the contribution. LGTM, but could you please update the README with the new config

**Comment by halkeye** [comment]
hopefully that works for you, i didn't want to add spacing to each other parameter

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ appVersion 1.16.0
 | `crPublish`         | `boolean` | `false` | `false`  | Enable chart-releaser publishing.                                                                                |
 | `crConfigPath`      | `string`  | `""`    | `false`  | Path to .ct.yaml chart-releaser configuration file.                                                              |
 | `isChartMuseum`     | `boolean` | `false` | `false`  | Enable ChartMuseum publishing.                                                                                   |
+| `populateChangelog` | `boolean` | `false` | `false`  | Populate `artifacthub.io/changes` annotations with notes produced by `@semantic-release/release-notes-generator` compatible plugins |
```
Comment (nflaig) [inline README.md:64]:
```suggestion
| `populateChangelog` | `boolean` | `false` | `false`  | Populate `artifacthub.io/changes` annotations with notes produced by `@semantic-release/release-notes-generator` compatible plugins. |
```

**Review by nflaig — APPROVED** [review]

**Comment by nflaig** [comment]
LGTM

**Comment by nflaig** [comment]
Thanks btw for the thorough testing 👍

**Comment by nflaig** [comment]
Published https://www.npmjs.com/package/semantic-release-helm3

**Comment by halkeye** [comment]
sweet, with this i can have version bumps when renovate updates the images 🤞

**Pull request merged**